LICENSES/Apache-2.0.txt

```diff
@@ -0,0 +1,73 @@
+Apache License
```

Comment:
add a github CI workflow to check the licenses like we have elsewhere

https://github.com/KDAB/cxx-qt/blob/2c319ec99f3f718d34c3a4ac271c79560797e637/.github/workflows/github-cxx-qt-tests.yml#L27-L32

Reply:
@LeonMatthesKDAB Also can you configure the repo so that it does not allow merge commits and requires CI to pass and CI has to run on the latest commit and approval etc (eg the usual settings)

Reply:
Done